apps/dashr-oil-and-gas-ternary/dataprep.R

```diff
@@ -0,0 +1,36 @@
+df <- read.csv("data/Well_ID-Table_1.csv")
+df[c(7,8,9,12,13,14)] <- lapply(df[c(7,8,9,12,13,14)], function(x) trimws(as.character(x)))
+df <- df[df['result'] == 'GAS',]
+df <- df[df['tdfm'] != '000',] 
+form_id <- unique(df['tdfm'])
+form_id <- form_id$tdfm
+form_id <- lapply(form_id, function(x) trimws(as.character(x)))
+
+
+df_fm <- read.csv("data/FmCodes-Table_1.csv")
+df_fm <- as.data.frame(lapply(df_fm, trimws))
+
+df['fm_name'] <- df['tdfm']
+df_fm <- df_fm[-c(1),] 
+
+formation_name <- list()
+i <- 1
```

Comment:
I found this interesting, looping using an index counter, I see that a lot in Python but not in R. It works fine, and I think it's more of a stylistic choice, personally I would iteratively loop through the range of elements in R, but I think it's your call how you do it. 👍

Reply:
for now, will stick with an index counter as it seems to work. definitely will use `range` or `seq` later on